README.md

```diff
@@ -1 +1,54 @@
-# corona-tastic
\ No newline at end of file
+# CoronaVirus Data
+
+> #### A web application in mobile form consuming up to date data on coronavirus by country and globally from an API and displaying it following a design on behance. [Original design](https://www.behance.net/sakwadesignstudio) idea by Nelson Sakwa.
+
```

Comment:
- [ OPTIONAL] [ ] It would be better if you include only the project description in the `description` section & gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the `Acknowledgement` section.